packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -419,7 +393,7 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 				}
 			: undefined;
 		return new BlobHandle(
-			`${BlobManager.basePath}/${id}`,
+			getGCNodePathFromBlobId(id),
```

Comment:
Surprising to see a GC-specific call here (at least by name).

Also... this seems to include a leading `/` which the old code doesn't.  Is that intentional?